CHANGELOG.md

```diff
@@ -24,7 +24,7 @@ compatible with the updates.
 
 ### Fixed
 
-- tba
+- Broken link in README to seminar video.
```

Comment:
Fixed the broken link lol.

Reply:
Does this need changing or are you approving as is

Reply:
Since the title is 'fixed' I assume that it is explanatory: fixed: The broken link in readme....